edit.html

```diff
@@ -391,6 +392,13 @@ <h1 id="heading">&nbsp;</h1> <!-- nbsp allocates the actual height which prevent
             </div>
           </div>
         </details>
+        <details id="integration" data-pref="editor.toc.expanded" class="ignore-pref-if-compact">
```

Comment:
You should add a new preference in prefs.js like `editor.integration.expanded` and use it here.